test/test_source_comment_associator.rb

```diff
@@ -2,18 +2,26 @@
 require 'parser/ruby18'
 
 class TestSourceCommentAssociator < Minitest::Test
-  def associate(code)
+  def associate(code, map_using_location=false)
     parser = Parser::Ruby18.new
 
     buffer = Parser::Source::Buffer.new('(comments)')
     buffer.source = code
 
     ast, comments = parser.parse_with_comments(buffer)
-    associations  = Parser::Source::Comment.associate(ast, comments)
+    if map_using_location
+      associations  = Parser::Source::Comment::Associator.new(ast, comments).associate_locations
```

Comment:
I think this is a good sign that `Parser::Source::Comment.associate` needs a keyword argument `using_locations: false`.